sdk/search/azure-search-documents/swagger/README.md

```diff
@@ -36,6 +36,40 @@ cd <swagger-folder>
 autorest --tag=searchindex
 autorest --tag=searchservice
 ```
+
+## Manual Changes
```

Comment:
Can some of these manual changes be done through customization?

Reply:
I think so, short term is get this documentation out there and we can begin working it back by using code generation customizations to reduce the amount of code we have to maintain